### PR TITLE
Add unit tests and CI

### PR DESCRIPTION
**Pull request by vladsavelyev**
* Add a dry test of the seqr loader workflow: `test_workflow_dry.py`
* Add test of parsing and checking QC: `test_check_multiqc.py`
* Add test of parsing and checking pedigree: `test_check_pedigree.py`
* Add Github CI

TODO in new PRs:
* Add individual jobs tests
* Add query code tests (annotation, vep)

### CODE REVIEW

File: test/test_check_multiqc.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+"PERCENT_DUPLICATION" = 25
+"""
+
+
+@pytest.fixture()
+def tmp_dir() -> Path:
+    dir_path = to_path('results') / timestamp()
+    dir_path.mkdir(parents=True, exist_ok=True)
+    yield dir_path
+    shutil.rmtree(dir_path)
```
Comment (MattWellie) [inline test/test_check_multiqc.py:32]: isn't this just replicating pytest's `tmpdir` fixture?

Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: Oh, didn't know about it! Thanks a lot @MattWellie  :D

Comment (MattWellie) [reply test/test_check_multiqc.py:32]: It's my favourite default fixture ❤️

Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: The only problem is that Hail's local backend doesn't work well with `/tmp`

Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: @MattWellie while you are here, would you please click "approve"?

Comment (MattWellie) [reply test/test_check_multiqc.py:32]: sample-metadata was removed from the requirements, which is killing the test runs.. should that be added back in?

Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: Oh, it actually should be `cpg-utils[workflows]` that installs `sample-metadata` now

**Review by MattWellie — APPROVED** [review]

File: test/test_check_multiqc.py (hunk 1 above)
Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: `cpg-utils` needs to be bumped :) https://github.com/populationgenomics/cpg-utils/pull/76

Comment (vladsavelyev) [reply test/test_check_multiqc.py:32]: Alright, tests are passing now, and the standard `tmpdir` fixture works here! Thanks @MattWellie :)

**Pull request merged**